.github/workflows/app-artifacts-win.yml

```diff
@@ -26,14 +35,43 @@ jobs:
       uses: crazy-max/ghaction-chocolatey@v1
       with:
         args: install make
+    - name: Azure CLI script
```

Comment:
suggestion: `name: Download certificates from Azure keyvault`

Link(s) to the relevant documentation sections could be helpful.